### DISCUSSION TITLE
【238320】【Outlast/逃生】【本体】

### DISCUSSION THREAD

**Discussion by chiaki-1999** [pjy612/SteamManifestCache#28] [Share]
code：FD0CAB6AE160248AD190369DAEED2C9411F0F87C

[Outlast.zip](https://github.com/pjy612/SteamManifestCache/files/13680577/Outlast.zip)

**Comment by chiaki-1999** [comment]
【1020790】【火影忍者 终极风暴羁绊】【本体】
[火影忍者 终极风暴羁绊.zip](https://github.com/pjy612/SteamManifestCache/files/13680641/default.zip)

**Reply by pjy612** [reply]
查收邮件
